### ISSUE TITLE
Quotes icon not working.

### ISSUE THREAD

**Issue by gatij10** [accordproject/web-components#255]
If the user highlights a paragraph or a group of paragraphs and clicks the quotes icons, the paragraphs are just indented, but not wrapped in quotes.
# Bug Report 🐛

## Expected Behavior

On clicking the quotes button, the text or selected paragraph should be wrapped inside quotes.

## Current Behavior

Below is link to the recorded video of the current behavior.
https://www.loom.com/share/8bcb3f2d4f8d4bc091176be1bf2f7263

**Comment by irmerk** [comment]
I believe this is a design choice in the Storybook (consumer of the `ui-markdown-editor`). We can adjust this in the props we give the editor in Storybook.

**Comment by d-e-v-esh** [comment]
Yes, but it looks a bit buggy. Isn't it?

**Comment by gatij10** [comment]
@irmerk 
I have opened a pull request. Let me know if there are any changes required.

**Comment by d-e-v-esh** [comment]
@gatij10  I did the exact same thing in my local app but seems like whatever I do from commenting out entire pages of code to deleting important lines does nothing to affect the actual running of the app. Am I doing something wrong? Do you have any idea what could be the problem?

**Comment by gatij10** [comment]
@d-e-v-esh you have to run the build command after you make the changes.

**Comment by d-e-v-esh** [comment]
@gatij10 
``lerna run build`` 
This command? Every single time?

**Comment by Michael-Grover** [comment]
I don't think there is a bug here. The existing behavior is intended. There should be no quotation marks around the text. People that use accord project components can customize the styling of the block quotes. For example, in Clause's implementation of the Accord Project Contract Editor, block quotes are rendered like this:

![image](https://user-images.githubusercontent.com/40569793/108895271-5e975700-75e1-11eb-9749-4b4c6367825d.png)

**Comment by gatij10** [comment]
@Michael-Grover the on clicking the quotes button, the text is just getting intended, even the vertical line is not visible.

**Comment by Michael-Grover** [comment]
I looked into different ways that documents show block quotes, and it looks like an indentation is sufficient to show a block quote. Some documents show block quotes with just an indentation, some use an indentation and a vertical line, some use an indentation and change the color of the text, etc. I think what we do currently works: going with the simplest option and allowing consumers of this component to customize the block quotes to appear the way they want.

**Comment by gatij10** [comment]
@Michael-Grover Okay.

**Comment by Michael-Grover** [comment]
@K-Kumar-01 yeah that sounds good, thanks

**Comment by K-Kumar-01** [comment]
> @K-Kumar-01 yeah that sounds good, thanks

@Michael-Grover 
I think you have wrongly tagged me. :grimacing:

**Comment by Michael-Grover** [comment]
@K-Kumar-01 yes that was a mistake. I've been bouncing around a bunch of different issues today.
